cfpb/consumerfinance.gov
# TDP: hardcode `TdpSearchHeroImage`

TDP has a `TdpSearchHeroImage` in Wagtail for setting the large and small hero image on https://www.consumerfinance.gov/consumer-tools/educator-tools/youth-financial-education/teach/activities/

In the history of that page, it doesn't appear we've ever changed the image. It seems like it would remove some Wagtail cruft to just hardcode the hero image. It can still be updated by updating the URLs in the template.

Also, the image didn't have an `alt` value, and this PR adds one.

## Removals

- TDP: hardcode `TdpSearchHeroImage`


## How to test this PR

1. `python cfgov/manage.py migrate` and visit http://localhost:8000/consumer-tools/educator-tools/youth-financial-education/teach/activities/ and compare to production and see that the hero image is unchanged.
